js/pta.js

```diff
@@ -0,0 +1,10 @@
+import AttorneyBanner from './components/attorney-banner/attorney-banner';
+
+function initAll() {
+  const attorneyBanners = document.querySelectorAll('[data-module="pta-attorney-banner"]');
+  attorneyBanners.forEach((attorneyBanner) => {
+    new AttorneyBanner(attorneyBanner).init();
+  });
+}
+
+initAll();
```

Comment:
@matthewmascord assets frontend has this run when stuffs finished loading rather than immediately, I'm guessing that's ok though since this js will be included at bottom of page rather than in the head, what do you think?

https://github.com/hmrc/assets-frontend/blob/e0aafb5b00d24db14671aaed317cec91bf6d130a/assets/javascripts/index.js

Reply:
this is the same as what we're doing in hmrc-frontend: https://github.com/hmrc/hmrc-frontend/blob/main/src/all-govuk-and-hmrc.js

Yep it should be loaded in hmrcScripts which is at the bottom of the page.